guides/wide-ep-lws/manifests/gateway/base/httproute.yaml

```
@@ -0,0 +1,26 @@
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: llm-route
```

It should convey that this route targets the deepseek inferencepool
```suggestion
  name: deepseek-r1
```